CONTRIBUTING.md

```diff
@@ -49,6 +49,20 @@ on how to start working on Servo.
 
 For specific git instructions, see [GitHub workflow 101](https://github.com/servo/servo/wiki/Github-workflow).
 
+## Running tests in pull requests
+
+When you push to a pull request, GitHub automatically checks that your changes have no compile, lint, or tidy errors.
```

Comment:
```suggestion
When you push to a pull request, GitHub automatically checks that your changes have no compilation, lint, or tidy errors.
```

Reply:
done in fddc8b2b8e30daa09ca92f435bedece7770fe8bb